### ISSUE TITLE
why mse > kl?

### ISSUE THREAD

**Issue by longlfuture** [HazyResearch/lolcats#16]
Thanks for your work!, since using kernel to fit the distribution of softmax, so why you choose MSE as loss function, instead of KL loss, can you give an explanation for this, thank you very much!
